test/e2e/sagemaker/tests/test_endpointConfig.py

```diff
@@ -0,0 +1,142 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
Names for test files in PyTest should be snake case (https://docs.pytest.org/en/reorganize-docs/new-docs/user/naming_conventions.html)